src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
+import NavBar from './components/navigation';
+import Books from './components/appBook';
+import Categories from './components/pageCat';
+import NotFound from './components/impossible';
+
+const App = () => (
+  <Router>
+    <NavBar />
+    <section className="content">
+      <Routes>
+        <Route path="/bookstore" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
+        <Route path="/*" element={<NotFound />} />
```

Comment:
- [ ] It would be better when you first visit the site if instead of showing the `Not found` component it shows the lists of books. It's really nice this feature but you are creating an extra page that you will not use and for your users could be confusing when starting the app don't find any content. Please fix this minor issue.  

```suggestion
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
```

Don't forget to change the path in your navbar 😄 👍

Reply:
I will do it 😄